packages/dropdowns/stories/examples/Menu/Default.tsx

```diff
@@ -0,0 +1,131 @@
+
+interface IStoryProps {
+  placement:
+    | 'auto'
+    | 'top'
+    | 'top-start'
+    | 'top-end'
+    | 'bottom'
+    | 'bottom-start'
+    | 'bottom-end'
+    | 'end'
+    | 'end-top'
+    | 'end-bottom'
+    | 'start'
+    | 'start-top'
+    | 'start-bottom';
+  hasArrow: boolean;
+  isAnimated: boolean;
+  isCompact: boolean;
+  isDisabled: boolean;
+}
+
+export const Default: Story<IStoryProps> = ({
+  hasArrow,
+  isAnimated,
+  isCompact,
+  placement,
+  isDisabled
+}) => {
+  return (
+    <Grid>
+      <Row style={{ minHeight: 200 }}>
+        <Col textAlign="center">
+          {/* eslint-disable-next-line no-alert */}
+          <Dropdown onSelect={item => alert(item)}>
+            <Trigger>
+              <Button>Default Menu</Button>
+            </Trigger>
+            <Menu
+              hasArrow={hasArrow}
+              isAnimated={isAnimated}
+              isCompact={isCompact}
+              placement={placement}
+            >
+              <Item disabled={isDisabled} value="option-1">
+                Option 1
+              </Item>
+              <Item disabled={isDisabled} value="option-2">
+                Option 2
+              </Item>
+              <Item disabled={isDisabled} value="option-3">
+                Option 3
+              </Item>
+            </Menu>
+          </Dropdown>
+        </Col>
+        <Col textAlign="center">
+          {/* eslint-disable-next-line no-alert */}
+          <Dropdown onSelect={item => alert(item)}>
+            <Trigger>
+              <IconButton aria-label="Settings" title="Settings">
+                <SettingsIcon />
+              </IconButton>
+            </Trigger>
+            <Menu
+              hasArrow={hasArrow}
+              isAnimated={isAnimated}
+              isCompact={isCompact}
+              placement={placement}
+            >
+              <Item disabled={isDisabled} value="option-1">
+                Option 1
+              </Item>
+              <Item disabled={isDisabled} value="option-2">
+                Option 2
+              </Item>
+              <Item disabled={isDisabled} value="option-3">
+                Option 3
+              </Item>
+            </Menu>
+          </Dropdown>
+        </Col>
+      </Row>
+    </Grid>
+  );
+};
+
+Default.argTypes = {
+  hasArrow: { name: 'hasArrow', control: 'boolean' },
+  isAnimated: { name: 'isAnimated', control: 'boolean' },
+  isCompact: { name: 'isCompact', control: 'boolean' },
+  placement: {
+    name: 'Placement',
+    control: {
+      type: 'select',
+      options: [
+        'auto',
+        'top',
+        'top-start',
+        'top-end',
+        'bottom',
+        'bottom-start',
+        'bottom-end',
+        'end',
+        'end-top',
+        'end-bottom',
+        'start',
+        'start-top',
+        'start-bottom'
+      ]
+    }
+  },
+  isDisabled: { name: 'Disabled items', control: 'boolean' }
+};
+
+Default.args = {
+  placement: 'bottom'
```

Comment:
Should this default placement be `bottom-start` to align with the `Menu` component?

Reply:
Updated, thanks 👍🏽 .